docs/shared/recipes/reduce-repetitive-configuration.md

```diff
@@ -0,0 +1,255 @@
+# Reduce Repetitive Configuration
+
+Nx can help you dramatically reduce the lines of configuration code that you need to maintain.
+
+Lets say you have three libraries in your repository - `lib1`, `lib2` and `lib3`. The folder structure looks like this:
+
+```treeview
+repo/
+├── libs/
+│   └── lib1/
+│   │   ├── tsconfig.lib.json
+│   │   └── project.json
+│   └── lib2/
+│   │   ├── tsconfig.lib.json
+│   │   └── project.json
+│   └── lib3/
+│       ├── tsconfig.lib.json
+│       └── project.json
+└── nx.json
+```
+
+## Initial Configuration Settings
+
+All three libraries have a similar project configuration. Here is what their `project.json` files look like:
+
+```json {% fileName="libs/lib1/project.json" %}
```

Comment:
Maybe we can use the "tabs" component here? Just to reduce the overall length of the doc and make it easier to parse. You could also highlight which sections we are talking about in particular. Because you can really only reduce the `options` prop here. Using the new highlight feature you could emphasize that in the code potentially.